usr/share/lib/img_proof/tests/SLES/SAP/test_sles_sap_hardened.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+
+
+@pytest.mark.skipinbeta
```

Comment:
why do we want to skip it in beta ? this means that we will not be able to test hardened images till the release

Reply:
I see the same decorator in lots of other tests, so I simply copied it here.

Reply:
this decorator will mute the test when in openQA side you run test  with `BETA=1` .

https://github.com/SUSE-Enceladus/img-proof/commit/af84ea206371a3bb96bd3759371606e11dd80ec4

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L201

ping me in slack if you still have questions

Reply:
Skip in beta is only for tests that are expected to fail during beta testing.